assertk/src/commonMain/kotlin/assertk/assert.kt

```diff
@@ -187,6 +187,7 @@ fun <T> Assert<T>.all(body: Assert<T>.() -> Unit) {
  *
  * @see assertFailure
  */
+@Deprecated("Use assertThat or assertFailure", ReplaceWith("assertThat(runCatching(f))"))
```

Comment:
A warning saying to use assertThat instead assertThat seems confusing? Maybe
```suggestion
@Deprecated("Use assertThat(result) or assertFailure", ReplaceWith("assertThat(runCatching(f))"))
```